app/config/security.yml

```diff
@@ -261,6 +266,7 @@ security:
         - { path: '^/espace-adherent/documents', roles: [ROLE_ADHERENT] }
         - { path: '^/espace-adherent/contacter', roles: [ROLE_ADHERENT] }
         - { path: '^/espace-adherent/mes-evenements', roles: [ROLE_ADHERENT] }
+        - { path: '^/espace-coordinateur/.*', roles: [ROLE_COORDINATOR] }
```

Comment:
wouldn't `path: '^/espace-coordinateur'` be enough ?

Reply:
yes, done 👍